tests/test_dcos_e2e/docker_utils/test_loopback.py

```diff
@@ -18,21 +18,34 @@ def test_loopback(self) -> None:
         client = docker.from_env(version='auto')
 
         with DockerLoopbackVolume(size=1) as device:
-            block_device_exists_cmd = ['lsblk', device.path]
             new_container = client.containers.create(
                 privileged=True,
                 detach=True,
                 image='centos:7',
             )
             new_container.start()
+
+            block_device_exists_cmd = ['lsblk', device.path]
             exit_code, output = new_container.exec_run(
                 cmd=block_device_exists_cmd,
             )
-            new_container.stop()
-            new_container.remove()
+            assert exit_code == 0, device.path + ': ' + output.decode()
 
+            block_device_has_right_size = [
```

Comment:
Optional change:

Instead use a command which prints the device size, then make the comparison of something like `assert output == '1048576'`. That will help us find out what the size actually is if this comparison fails.